projects/graph/src/graph.py

```diff
@@ -74,8 +75,11 @@ def bfs(self, starting_point, value):
             return True
         return False
 
-    def dfs(self):
-        pass
+    def dfs(self, starting_point, value):
+        a = self.dfts(starting_point)
```

Comment:
this should be able to find a unconnected vertex if I start at it. currently doesn't.